R/save_arcadia_plot.R

```diff
@@ -0,0 +1,44 @@
+#' Save Arcadia Plot
+#'
+#' @param plot ggplot object to be saved
+#' @param filepath The path where the plot will be saved
+#' @param orientation The orientation of the plot, either "landscape" or "portrait"
+#'
+#' @return None
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' p <- ggplot(mtcars, aes(x = hp, y = mpg)) + geom_point()
+#' save_arcadia_plot(p, "plot.pdf", "landscape")
+#' }
+save_arcadia_plot <- function(filepath, plot, orientation = c("landscape", "portrait")) {
+  # Ensure the orientation argument is valid
+  orientation <- match.arg(orientation)
+
+  # Set size based on orientation
+  if (orientation == "landscape") {
+    width <- 11
+    height <- 8
+  } else if (orientation == "portrait") {
+    width <- 8
+    height <- 11
+  }
+
+  # Use ggsave to save the plot
+  ggplot2::ggsave(
+    filename = filepath,
+    plot = plot,
+    device = "pdf",
+    dpi = 300,
```

Comment:
How does this dpi affect the relative sizing of fonts? When opening the PDFs in Illustrator, do you see the right font sizes still? I had to save at 72 dpi in order to get things to look correct in `matplotlib`.

Reply:
I think the font sizes look ok? I can change to 72 DPI if that's what you have.

Reply:
I changed to 72 DPI, it doesn't look different to me I think?

Reply:
I think the real test would be to try to drag and drop the plots into the current Illustrator templates and see whether they fit. Not critical to get this done by Tuesday though.